### PR TITLE
Strong name sign Migration Library

### PR DESCRIPTION
**Pull request by brthor**
@333fred @eerhardt 

Strong name sign migration library

### CODE REVIEW

**Comment by eerhardt** [comment]
Looks good - but it is still red....

**Comment by brthor** [comment]
Hmmm looks like I need to strong name the tests as well.

**Comment by 333fred** [comment]
Looks like something with InternalsVisibleTo isn't working anymore

File: src/Microsoft.DotNet.ProjectJsonMigration/project.json [hunk 1]
```diff
@@ -1,7 +1,8 @@
 ﻿{
   "version": "1.0.0-preview3-*",
   "buildOptions": {
-    "warningsAsErrors": true
+    "warningsAsErrors": true,
+    "keyFile": "../../tools/Key.snk"
```
Comment (333fred) [inline src/Microsoft.DotNet.ProjectJsonMigration/project.json]: Will this make local builds public-signed?

Comment (brthor) [inline src/Microsoft.DotNet.ProjectJsonMigration/project.json]: yes

**Comment by brthor** [comment]
The tests use a different key file, and I had some namespace issues in migration

**Comment by brthor** [comment]
test Windows_NT x64 Release Build

**Comment by brthor** [comment]
test OSX x64 Release Build

**Pull request merged**